backend/kesaseteli/settings.py

```diff
@@ -1,6 +1,5 @@
-import os
-
 import environ
+import os
```

Comment:
Your isort config is incorrect, since `os` is a standard library, but `environ` is a third party library so they should be in different sections.